src/catalog/src/builtin.rs

```diff
@@ -6725,6 +6728,42 @@ pub const MZ_INTROSPECTION_CLUSTER_REPLICA: BuiltinClusterReplica = BuiltinClust
     cluster_name: MZ_INTROSPECTION_CLUSTER.name,
 };
 
+pub const MZ_PROBE_CLUSTER: BuiltinCluster = BuiltinCluster {
+    name: "mz_probe",
+    owner_id: &MZ_SYSTEM_ROLE_ID,
+    privileges: &[
+        MzAclItem {
+            grantee: MZ_SUPPORT_ROLE_ID,
+            grantor: MZ_SYSTEM_ROLE_ID,
+            acl_mode: AclMode::USAGE,
+        },
+        MzAclItem {
+            grantee: MZ_MONITOR_ROLE_ID,
+            grantor: MZ_SYSTEM_ROLE_ID,
+            acl_mode: AclMode::USAGE,
+        },
+        rbac::owner_privilege(ObjectType::Cluster, MZ_SYSTEM_ROLE_ID),
+    ],
+};
+// maybe we can get rid of replicas in this and just use a size/
```

Comment:
This seems fine to me, even for the long term. Probably not worth the comment unless you're planning to address imminently!

Reply:
Yeah note to self I hadn't cleaned up.